docs/reference/api.md

```diff
@@ -272,7 +272,7 @@ addStuff().then((d) => console.log(d)); // 13 (5 + 7 + 1)
 
 | Name                                   | Example                                              | Description                                                                                                                                                                                    |
 | --------------------------------- | --------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `accessNode.api` **(required)**        | `https://rest-testnet.onflow.org` (testnet)<br />`https://rest-mainnet.onflow.org` (mainnet)                  | API URL for the Flow Blockchain Access Node you want to be communicating with. For more information, see [here](https://docs.onflow.org/fcl/reference/sdk-guidelines/#connect) |
+| `accessNode.api` **(required)**        | `https://access-testnet.onflow.org`                  | API URL for the Flow Blockchain Access Node you want to be communicating with. See all available access node endpoints [here](https://docs.onflow.org/access-api/#flow-access-node-endpoints). |
```

Comment:
Should be `https://rest-testnet.onflow.org` as HTTP is now default in FCL ^1.0.0